Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
Yeah my concern is primarily with keeping the `:branch => "foo"`, which is required whenever "foo" != "master", on the same line as the `url "https://github.com/this/that.git",` even when it spills off the page.